pageserver/src/layered_repository/storage_layer.rs

```diff
@@ -91,6 +92,11 @@ pub trait Layer: Send + Sync {
     /// Range of keys that this layer covers
     fn get_key_range(&self) -> Range<Key>;
 
+    /// Get layer creation time
+    fn get_creation_time(&self) -> Result<SystemTime> {
```

Comment:
Probably worth a comment that this time should not be relied upon too much. The same reason as is here: https://github.com/zenithdb/zenith/pull/1332#discussion_r817948306

Reply:
Done